background/style-via-api.js

```diff
@@ -56,6 +56,9 @@
       return NOP;
     }
     return API.styles.getSectionsByUrl(url, id).then(sections => {
+      if (sections.cfg) {
```

Comment:
What about removing the condition so only `delete sections.cfg` remains? It's not a complex operation to warrant the check. Also in [apply.js](https://github.com/openstyles/stylus/pull/1364/files#diff-ce566497474525b9fe6be695bb4f12140dc22803c52c93703f1467db27ac1030R190).